4.4.1.0/Dockerfile

```diff
@@ -13,4 +13,4 @@ RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E03280
 RUN echo "deb http://download.mono-project.com/repo/debian wheezy/snapshots/4.4.1.0 main" > /etc/apt/sources.list.d/mono-xamarin.list \
   && apt-get update \
   && apt-get install -y mono-devel ca-certificates-mono fsharp mono-vbnc nuget referenceassemblies-pcl \
-  && rm -rf /var/lib/apt/lists/*
+  && rm -rf /var/lib/apt/lists/* /var/cache/* /tmp/*
```

Comment:
`/var/cache/` is just 1MB on my 4.4.0 Docker image, I don't think we need to clean it.